test/parallel/test-util-inspect.js

```diff
@@ -296,7 +296,7 @@ assert.strictEqual(
 
 // Function with properties
 {
-  const value = function() {};
+  const value = () => {};
```

Comment:
This is another place where `common.noop` usage can lead to subtle bugs. The next line adds a property to the object, which would end up adding the property to `common.noop` which means possible side effects in other tests.